### PR TITLE
Build Chat Worktree actions and shared branch suggestions

### PR DESCRIPTION
**Pull request by Nek-12**
## Summary

Build the Chat Worktree sidebar actions: Switch, destination-local Create with shared submit shortcuts, and anchored Delete confirmation. Keep server-projected authority, Query-owned mutation completion, and visible-list-only refresh. Share the existing Go branch-name sanitizer between TUI and the server-provided nullable suggestion.

Addresses product task **KENT-396**, including the approved combined KENT-394/KENT-395 scope. No linked GitHub issue was supplied.

## QA verification

QA approved after re-verification in an isolated fixture-only browser mount of the real components and typed API client:

- Direct Switch internal failure displayed one dismissible notification with the supplied diagnostic instead of an RPC method/code.
- Automatic Switch failure after Create retained the form, value, and classification, with one failure notification.
- Earlier QA verified topology/action visibility, clean deletion preview, hierarchical Escape, local validation, classification, whitespace edits during and after resolution, focus return, narrow layout, and light/dark presentation.

The temporary mount, browser window, Vite process, and isolated state were removed. Attached screenshots are QA artifacts, not shipped screens or a production Chat integration.

## Diff size

Tracked final diff against `origin/main`, counting additions and deletions (not cumulative commit churn):

| Category | Added | Removed | Net | Gross |
| --- | ---: | ---: | ---: | ---: |
| Production code | +1,035 | -138 | +897 | 1,173 |
| Tests/generated/docs tracked in Git | +70 | -58 | +12 | 128 |
| Total tracked | +1,105 | -196 | +909 | 1,301 |

Generated Go/TypeScript contracts are ignored by repository policy. The implementation-stage generation measurement adds +30/-9 (net +21, gross 39) outside the tracked diff. Including that measurement, tests/generated/docs total +100/-67 (net +33, gross 167), and the total delivery is +1,135/-205 (net +930, gross 1,340). The approved production budget conservatively includes these generated lines: 1,212, below 1,700.

## Caveats and follow-up

- Production Chat activation/composer integration remains separate. No showcase or new fixture subsystem ships; new surface QA used temporary mounts.
- Still unverified manually: Pending/Scheduled acknowledgements and later outcomes; dirty/unknown/missing/detached deletion variants and races; refresh/retry/cleanup feedback; setup-retained and dismissed-operation disposal; reconnect; Windows/Linux shortcuts; reduced motion; and complete platform shortcut coverage. Follow-up QA should cover these before broader activation.
- Ordinary Atom disposal timing remains the approved observation boundary. Dismissal does not cancel submitted mutations or required automatic Switch. Failed automatic Switch keeps the current creation form open.
- No connection-loss action gate was added; KENT-659 owns that broader policy.
- Review/QA uncovered same-key whitespace observation disposal and loss of typed error diagnostics; both were corrected. Blocker/capacity copy makes no unsupported claims about the cause.
- Publication rebased onto current `origin/main`. Conflicts were resolved by preserving the new Chat prompt-picker export, wiring Worktree translations through the extracted `chatEn` owner, and advancing protocol 160 to 161. No post-rebase runtime verification was performed in this publication-only phase.

Direct Switch failure with authoritative diagnostic

Automatic Switch failure re-verification

Clean deletion preview

Narrow creation form

## Summary by CodeRabbit

- **New Features**
  - Added a worktree creation flow with branch suggestions, target classification, base-branch selection, validation, and retry support.
  - Added worktree deletion confirmation, including options to retain or remove the associated branch.
  - Added clearer status messages for switching, creation, deletion, and worktree conditions.

- **Bug Fixes**
  - Worktree lists now refresh after actions when the relevant list is open.
  - Failed worktree operations now display localized, actionable error messages.
  - Creation drafts reset appropriately when leaving and reopening the form.

## PR review disposition

The human owner explicitly rejected the three additional reviewer requests for this PR: direct-create preload (separate command integration), extra action-boundary tests (outside the approved testing scope), and a whitespace-only internal-cause fallback. All three threads have explanatory replies and are resolved. No source changes were made; whitespace-only supplied causes can still appear blank and are not claimed fixed. Diff accounting and attached evidence are unchanged.

### CODE REVIEW

File: apps/desktop/src/app-facade/sidebarContext.ts [hunk 1]
```diff
@@ -55,7 +54,7 @@ export type SidebarDestination =
       kind: "worktree";
       mode?: SidebarMode;
       sessionID: string;
-      onAction: WorktreeBrowserActions;
+      page: "list" | "create";
```
[the remark this answers is not in the document]
Comment (Nek-12) [reply apps/desktop/src/app-facade/sidebarContext.ts:57]: The human owner explicitly chose to reject this change for this PR. Direct /worktree create integration belongs to KENT-397; this delivery enters Create from the Worktree list and keeps production Chat/command activation separate. No preload path is added here.

File: apps/desktop/src/features/chat/WorktreeBrowser.test.tsx [hunk 2]
```diff
@@ -324,47 +347,26 @@ it.each([
...
-    );
-    const projection = list?.worktrees[0]?.projection;
-    if (projection === undefined) throw new Error("Missing fixture projection");
-    const user = userEvent.setup();
-    await user.click(screen.getByText(kind === "mainWorkspace" ? "Workspace" : "Feature"));
-    expect(onAction).not.toHaveBeenCalled();
     const switchButtons = screen.queryAllByRole("button", { name: appI18n.t("chat.worktree.switch") });
     const deleteButtons = screen.queryAllByRole("button", { name: appI18n.t("chat.worktree.delete") });
     expect(switchButtons).toHaveLength(switches);
     expect(deleteButtons).toHaveLength(deletes);
```
[the remark this answers is not in the document]
Comment (Nek-12) [reply apps/desktop/src/features/chat/WorktreeBrowser.test.tsx:369]: The human owner explicitly chose to reject this change for this PR. The approved testing scope removes new UI/action tests and retains baseline coverage with interface adaptations. No additional action-boundary assertions will be added in this delivery.

File: apps/desktop/src/features/chat/worktreeErrorMessage.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import type { TFunction } from "i18next";
+
+import { errorMessage, SelectorErrorKind, WorktreeError } from "@/api";
+
+export function worktreeErrorMessage(error: unknown, t: TFunction): string {
+  if (!(error instanceof WorktreeError)) return errorMessage(error);
+  const detail = error.detail;
+  switch (detail.kind) {
+    case "internal":
+      return detail.cause ?? t("chat.worktree.internalFailure");
```
[the remark this answers is not in the document]
Comment (Nek-12) [reply apps/desktop/src/features/chat/worktreeErrorMessage.ts:10]: Presented the whitespace-only diagnostic edge case and proposed fallback change to the human owner. They explicitly chose to reject this change for this PR. Supplied diagnostic presentation remains unchanged; this is not claimed to be fixed.
